### PR TITLE
add Scribunto and VE to rawdatawiki

### PR DESCRIPTION
**Pull request by Reception123**

### CODE REVIEW

**Comment by revi** [comment]
Blocked by miraheze/puppet#19

File: LocalSettings.php [hunk 1]
```diff
@@ -275,9 +275,11 @@
 		'extloadwiki' => true,
 		'kurumiwiki' => true,
 		'quantixwiki' => true,
+		'rawdatawiki' => true,
```
Comment (Southparkfan) [inline LocalSettings.php]: Defined twice.

**Comment by Reception123** [comment]
@Southparkfan problem solved.

**Comment by Reception123** [comment]
@revi parsoid was merged

**Pull request merged**